src/ontology/mondo-ingest.Makefile

```diff
@@ -275,3 +274,32 @@ mappings/mondo-sources-all-lexical.sssom.tsv: $(SCRIPTSDIR)/match-mondo-sources-
 	# The $^ includes the python script, basically all paramters after the colon :
 
 lexical_matches: mappings/mondo-sources-all-lexical.sssom.tsv
+
+#############################
+###### Slurp pipeline #######
+#############################
+slurp/:
+	mkdir -p $@
+
+# TODO: Move this to Makefile. how, when I shouldn't edit it from this repo?
+.PHONY: component-download-mondo.owl
+component-download-mondo.owl: | $(TMPDIR)
+	if [ $(MIR) = true ] && [ $(COMP) = true ]; then $(ROBOT) merge -I http://purl.obolibrary.org/obo/mondo.owl \
+	annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) -o $(TMPDIR)/$@.owl; fi
+
+# TODO: Move this to Makefile. how, when I shouldn't edit it from this repo?
+$(COMPONENTSDIR)/mondo.owl: component-download-mondo.owl
+	if [ $(COMP) = true ] ; then if cmp -s $(TMPDIR)/component-download-mondo.owl.owl $@ ; then echo "Component identical."; else echo "Component is different, updating." && cp $(TMPDIR)/component-download-mondo.owl.owl $@; fi; fi
+
+# min-id: the next available Mondo ID
+slurp/%.tsv: $(COMPONENTSDIR)/%.owl $(TMPDIR)/mondo.sssom.tsv $(REPORTDIR)/mirror_signature-mondo.tsv | slurp/
+	python $(SCRIPTSDIR)/migrate.py \
+	--ontology-path $(COMPONENTSDIR)/%.owl \
+	--sssom-map-path $(TMPDIR)/mondo.sssom.tsv \
+	--min-id 123000 \
+	--mondo-terms-path $(REPORTDIR)/mirror_signature-mondo.tsv \
+	--outpath $@
+
+slurp-%: slurp/%.tsv
+
+slurp-all: slurp-omim slurp-doid slurp-ncit slurp-ordo slurp-icd10cm slurp-icd10who
```

Comment:
Added to `.PHONY`
`slurp-all`: I appended this to `.PHONY` at the top of the file. I think this is correct, even though ultimately it does end up creating files; just none with the name "slurp-all".
`slurp-%`: I wasn't going to append this to `.PHONY`, but then I realized that he same logic for "slurp-all" should apply here.

Reply:
Its correct!